test/myxql_test.exs

```diff
@@ -70,6 +70,14 @@ defmodule MyXQLTest do
       MyXQL.prepare_execute!(conn, "", "SELECT 1, 2, NOW()")
     end
 
+    test "custom socket options with :inet6" do
+      opts = [socket_options: [:inet6]] ++ @opts
```

Comment:
I am not sure if CI will like an IPv6 connection. If so, you can use `:inet` instead (for IPv4).

Reply:
```suggestion
      opts = [socket_options: [:inet]] ++ @opts
```